aha-table.html

```diff
@@ -179,6 +179,12 @@
         type: Boolean,
         value: false
       },
+      //greedy: if table needs to be greedy
```

Comment:
Please be more descriptive. Follow pattern of other properties. `greedy` isn't clear enough to someone without context.